robocop/checkers/naming.py

```diff
@@ -246,14 +249,14 @@ def visit_Keyword(self, node):  # noqa
         self.generic_visit(node)
 
     def visit_KeywordCall(self, node):  # noqa
+        if self.in_if_block and node.keyword and node.keyword.lower() in self.else_if:
```

Comment:
I know it was created earlier but maybe we can also rename this `self.else_if`? What about `else_statements` or `else_variants`? `else_if` is not explanatory.

Reply:
I will go with `else_statements`